src/test/java/org/fundacionjala/coding/cesar/EanTest.java

```diff
@@ -0,0 +1,45 @@
+package org.fundacionjala.coding.cesar;
+import org.junit.Before;
+import static org.junit.Assert.assertEquals;
+import org.junit.Test;
+
+/**
+ *
+ * @author admin-hp
+ */
+public class EanTest {
+    private Ean val;
+
+    /**
+     * Starting the before annotation.
+     */
+    @Before
+    public void before() {
+        val = new Ean();
+        System.out.println("before");
+    }
+
+    /**
+     * test for Validar pass.
+     */
+    @Test
+    public void testValidarPass() {
+        System.out.println("pasa");
+        boolean res;
+        val.setCadena("4003301018398");
+        res = val.validar();
+        assertEquals(true, res);
+    }
+
+    /**
+     * test for Validar failed.
+     */
+    @Test
+    public void testValidarFailed() {
+        System.out.println("no pasa");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)